shared/common-adapters/plain-input.d.ts

```diff
@@ -83,6 +83,7 @@ export type Props = {
   placeholderColor?: string
   rowsMin?: number
   rowsMax?: number
+  secureTextEntry?: boolean
```

Comment:
I don't think this is controlling anything, add a `|| this.props.secureTextEntry` here?

https://github.com/keybase/client/blob/08eec9bc220d2348330b8756f55fbd4d11be4bc5/shared/common-adapters/plain-input.native.tsx#L231

Reply:
added!